### PR TITLE
Change template design for checkout blockonomics

### PR DESCRIPTION
**Pull request by aadyanthaya**
Made changes in the Blockonomics checkout template preview 

Web View  | Mobile view
---- | --- 
image|image
image|image
image|image
image|image

### CODE REVIEW

**Comment by blockonomics** [comment]
Please use updated https://blockonomics.notion.site/Modular-Checkout-Wordpress-85eea8114c1e4fc4bd2af8ade3bfa422

**Comment by aadyanthaya** [comment]
image

image


Theme Twenty Twenty-One

**Comment by aadyanthaya** [comment]
image
image


theme:twenty twenty-two

**Comment by aadyanthaya** [comment]
image
image


theme:Omega

**Comment by aadyanthaya** [comment]
image
image

Theme: Custom

**Comment by shivaenigma** [comment]
Lets remove the  |OR  in both mobile and desktop. Design view it seems to look ok, but now it doesn't seem necessary

File: css/order.css [hunk 1]
```diff
@@ -398,3 +364,191 @@ svg {
...
+.bnomics-order-panel {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.bnomics-header-container,
```
Comment (DarrenWestwood) [inline css/order.css:425]: These CSS changes appear to be unnecessary and have little to no affect. Let's ensure we are not including CSS changes that are not required.
We also should not be hardcoding font-size's. We can consider using a relative font size, although this does not appear necessary.

File: css/order.css [hunk 2]
```diff
@@ -398,3 +364,191 @@ svg {
   text-align: center;
   transition: 300ms all ease-in-out;
 }
+
+#bnomics-address-input,
+#bnomics-amount-input {
+  padding: 5px;
+  max-width: 190%;
```
Comment (DarrenWestwood) [inline css/order.css:371]: `max-width: 190%;` does not appear to be necessary. Also `max-width: 580px;` below.

File: css/order.css [hunk 3]
```diff
@@ -398,3 +364,191 @@ svg {
...
+  display: none;
+}
+
+table.blockonomics_checkout_table tbody td {
+  background-color: transparent !important;
+}
+
+
+
+@media screen and (max-width: 600px) {
```
Comment (DarrenWestwood) [inline css/order.css:479]: At 600px, there is no border for both the outer table td's and the inner table td's.

File: templates/blockonomics_checkout.php [hunk 4]
```diff
@@ -61,54 +64,70 @@
...
                 <tr>
-                    <th>
-                        <label class="bnomics-amount-text"><?= __('Amount of', 'blockonomics-bitcoin-payments') ?> <?php echo strtolower($crypto['name']); ?> (<?php echo strtoupper($crypto['code']); ?>) <?= __('to send:', 'blockonomics-bitcoin-payments') ?></label>
-                        <label class="bnomics-copy-amount-text"><?= __('Copied to clipboard', 'blockonomics-bitcoin-payments') ?></label>
-
-                        <div class="bnomics-copy-container" id="bnomics-amount-copy-container">
-                            <input type="text" value="<?php echo $order_amount; ?>" id="bnomics-amount-input" readonly />
-                            <span id="bnomics-amount-copy" class="blockonomics-icon-copy"></span>
-                            <span id="bnomics-refresh" class="blockonomics-icon-refresh"></span>
+                    <th class="bnomics-footer-container">
```
[the remark this answers is not in the document]
Comment (aadyanthaya) [reply templates/blockonomics_checkout.php:118]: changes

**Review by DarrenWestwood — COMMENTED** [review]
Changes look good and almost ready for merge. 2 minor changes:
1. Price refresh is not working after recent changes due to this line: https://github.com/blockonomics/woocommerce-plugin/blob/d8fc94c9a5f5f8827c50567c753deec2613d25a6/js/checkout.js#L292

**Pull request merged**